plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizardPage.java

```diff
@@ -270,12 +270,13 @@ protected IStatus validate() {
 
 		// connect error
 		dbc.addValidationStatusProvider(new MultiValidator() {
+			IObservableValue observable = BeanProperties
+					.value(ConnectionWizardPageModel.PROPERTY_CONNECTED_STATUS, IStatus.class)
+					.observe(pageModel);
```

Comment:
oh, good catch! The prior code was creating a new observable in each validation pass (which there are many).